Makefile

```diff
@@ -7,13 +6,6 @@ SCHEDULER_PID=airflow-scheduler.pid
 WORKER_PID=airflow-worker.pid
 FLOWER_PID=airflow-flower.pid
 
-init:
-	pyenv global $(PYTHON_VERSION)
-	pyenv install ${PYTHON_VERSION}
-	pyenv virtualenv ${PYTHON_VERSION} workflows
-	pyenv activate workflows
-	export AIRFLOW_HOME=${PWD}
```

Comment:
Why removing it? We just need to update it and to align it with the documentation

Reply:
I understood wrong: we should keep the python version installation in the documentation
That we should keep just in documentation